tests/cases/fourslash/inlayHintsShouldWork49.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path="fourslash.ts" />
+
+// @allowJs: true
+// @checkJs: true
+
+// @Filename: /a.js
+//// /**
+////  * @type {string}
+////  */
+//// var x = ""
+
+
+goTo.file('/a.js')
+const markers = test.markers();
+verify.getInlayHints([], undefined, {

```

Comment:
what does this test do?

Reply:
It's means 'No hints provided'. (as well as the above one)